split-rum/manifest.json

```diff
@@ -0,0 +1,43 @@
+{
+  "manifest_version": "2.0.0",
+  "app_uuid": "TODO:WHAT_IS_THE_UUID_FOR_THIS_ONE?",
+  "app_id": "split-rum",
+  "display_on_public_website": true,
+  "tile": {
+    "overview": "README.md#Overview",
+    "configuration": "README.md#Setup",
+    "support": "README.md#Support",
+    "changelog": "CHANGELOG.md",
+    "description": "Enriches your RUM data with your feature flags from Split",
+    "title": "Split",
+    "media": [
+      {
+        "caption": "Datadog RUM with Split Feature Flags overview (list of flags)",
```

Comment:
I noticed a slight typo in this image - inferred has an extra 'r' :)

Reply:
Good catch.  Thank you!
I fixed this in latest version of split-rum/images/split-feature-flags-list-in-dd-rum.png committed a moment ago.